Pronto-MIA/Pronto-MIA/DataAccess/Managers/DeploymentPlanManager.cs

```diff
@@ -134,11 +152,27 @@ public class DeploymentPlanManager : IDeploymentPlanManager
         }
 
         /// <inheritdoc/>
-        public IQueryable<DeploymentPlan?> GetAll()
+        public IQueryable<DeploymentPlan> GetAll()
         {
             return this.dbContext.DeploymentPlans;
         }
 
+        /// <inheritdoc/>
+        public async Task<DeploymentPlan> GetById(int id)
```

Comment:
Is there a specific reason why `GetAll()` is sync and `GetById()` is async?

Reply:
GetAll returns a dbContext queryable which can then be queried. The query operation on the context itself is what is async so the async operation is pushed to the caller when using `GetAll` whilst `GetById` already calls the async operation.